src/components/App.js

```diff
@@ -40,10 +42,10 @@ const HomeRouter = () => (
       <>
         <Route component={MusicMarkdownNavbar} />
         <Switch>
-          <Route path={`${REPO_REGEX}/viewer/:branch/:path+`} component={MarkdownViewer} />
-          <Route path={`${REPO_REGEX}/browser/:branch/:path*`} component={RepositoryNavigation} />
-          <Route path={`${REPO_REGEX}/editor/:branch/:path*`} component={MarkdownEditor} />
-          <Route path={REPO_REGEX} component={BranchNavigation} />
+          <Route path={`/viewer/${REPO_REGEX}/:branch/:path+`} component={MarkdownViewer} />
+          <Route path={`/browser/${REPO_REGEX}/:branch/:path*`} component={RepositoryNavigation} />
+          <Route path={`/editor/${REPO_REGEX}/:branch/:path*`} component={MarkdownEditor} />
+          <Route path={`/${REPO_REGEX}/branches`} component={BranchNavigation} />
```

Comment:
This can be simplified with your new routing scheme. Make the Route path match `/browser/${REPO_REGEX}` (no postfix) to take you to the Branch Navigation.